### ISSUE TITLE
aws_security_group_rule.ingress_rules[1]: : invalid or unknown key: description

### ISSUE THREAD

**Issue by Vad1mo** [terraform-aws-modules/terraform-aws-security-group#29]
I now (1.9.0) get this error: `aws_security_group_rule.ingress_rules[1]: : invalid or unknown key: description`

However the setup was working before
```
module "wvm_ssh_sg" {
  source = "terraform-aws-modules/security-group/aws"
  version = "1.9.0"


  name        = "wvm default"
  description = "Default Security group for the workshop VMs. with SSH,HTTP/S and RDP"
  vpc_id      = "${module.vpc.vpc_id}"

  ingress_cidr_blocks      = ["0.0.0.0/0"]
  ingress_rules            = ["ssh-tcp","http-80-tcp","https-443-tcp"]  
  ingress_with_cidr_blocks = [{
     from_port   = 3389
     to_port     = 3389
     protocol    = "tcp"
     cidr_blocks = "0.0.0.0/0"
     },{
     from_port   = 3389
     to_port     = 3389
     protocol    = "udp"
     cidr_blocks = "0.0.0.0/0"
     },
   ]
  ingress_ipv6_cidr_blocks = ["::/0"]

  #egress_cidr_blocks = []
  #egress_ipv6_cidr_blocks = []
  #egress_rules = ["all-all"]
  egress_with_cidr_blocks = [
    {
      rule             = "all-all"
      cidr_blocks      = "0.0.0.0/0"
      ipv6_cidr_blocks = "::/0"
    },]
}
```

I am using terraform 0.11

**Comment by Sashan** [comment]
It looks like I'm hitting same problem, may I ask you how did you solve it?
thanks a lot.

**Comment by Vad1mo** [comment]
updated tf downloaded the modules again, it started working again.

**Comment by antonbabenko** [comment]
[resource/aws_security_group: Allow empty rule description](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1100-february-24-2018) was fixed in AWS provider version 1.10.0 which was released 2 weeks ago. This was probably the solution which worked.
